stdlib/public/core/Integers.swift.gyb

```diff
@@ -2320,24 +2292,24 @@ ${unsafeOperationComment(x.operator)}
   }
 % end
 
-  @inline(__always)
+  @_transparent
```

Comment:
Yep; I made this change at the last minute to work around a weird performance regression. I should probably change it back.